hasher-matcher-actioner/webapp/public/index.html

```diff
@@ -11,9 +11,22 @@
       .switch-wrapper > div {
         position: absolute;
       }
+
```

Comment:
Having this `<style>` element here inline in our main HTML entry point is a turd left over from my rushing through a first hack at things. Can you create a main .css file, reference it here and move the css out to there?

Reply:
Sure!